star-wars-app/Pages/People/PeopleView.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  PeopleView.swift
+//  star-wars-app
+//
+//  Created by Michał Nowak on 13/01/2020.
+//  Copyright © 2020 Michal Nowak. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class PeopleView: UIView {
+    let tableView = UITableView()
+    let progressLabel = UILabel()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+    
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+private extension PeopleView {
+    func setup() {
+        setupViewHierarchy()
+        applyStyling()
+    }
+    
+    func setupViewHierarchy() {
+        setupProgressView()
+        setupTableView()
+    }
+    
+    func applyStyling() {
+        progressLabel |> Label.setText("Loading..")
+        tableView |> View.negateHidden
+    }
+    
+    private func setupTableView() {
+        addSubview(tableView)
+        
+        tableView |> View.notTranslatesAutoresizing
+        
+        addConstraints([
+            tableView.topAnchor.constraint(equalTo: layoutMarginsGuide.topAnchor),
+            tableView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            tableView.trailingAnchor.constraint(equalTo: trailingAnchor),
+            tableView.bottomAnchor.constraint(equalTo: bottomAnchor),
+        ])
+        
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: "peopleCellId")
```

Comment:
1. This part I'd actually do in view controller, since when you'll be changing something, like adding a new kind of cell or changing this one, it's better to have both in a single place.

2. We try to avoid hardcoding these kinds of strings, since it's easy to make a typo and it won't work. If you need to write it down in code, it's better to store it some constant in order to be sure you're using the same thing everywhere.
We tend to use this little library in our projects: https://github.com/AliSoftware/Reusable/blob/master/Sources/View/Reusable.swift
You can check out the implementation and even copy the part that you need to your project, since it's super simple and can give you the idea how to use similar concept in other places.